app/views/clients/clients.html

```diff
@@ -30,8 +30,9 @@
         </tr>
         </tbody>
     </table>
-    <ul class="pager">
+    <!-- <ul class="pager">
         <li class="previous"><a id="prev" ng-click="clients.previous()" href="" ng-disabled="!clients.hasPrevious()">&larr; {{'label.button.previous' | translate}}</a></li>
         <li class="next"><a id="next" ng-click="clients.next()" href="" ng-disabled="!clients.hasNext()">{{'label.button.next' | translate}} &rarr;</a></li>
-    </ul>
+    </ul> -->
+    <dir-pagination-controls template-url="../../bower_components/angular-utils-pagination/dirPagination.tpl.html" on-page-change="getResultsPage(newPageNumber)"></dir-pagination-controls>
```

Comment:
template url etc should be set globally looks like https://github.com/michaelbromley/angularUtils/tree/master/src/directives/pagination#specifying-the-template provides details for the same